dataFields/ABFieldDateCore.js

```diff
@@ -9,7 +9,10 @@ const ABField = require("../../platform/dataFields/ABField");
 
 function L(key, altText) {
```

Comment:
Should it call `AB.Label()` to support multilingual ?

Reply:
Call AB.Label() replace function L